frontend/lib/form-input-converter.ts

```diff
@@ -40,10 +40,7 @@ function assertNullOrBool(key: string|number|symbol, value: unknown): null|boole
  * Convert the given value to a boolean, assigning a default if it's `null`.
  */
 function nullToBool(value: boolean|null, defaultValue: boolean): boolean {
-  if (value === null) {
-    return defaultValue;
-  }
-  return value;
+  return value ?? defaultValue;
```

Comment:
Nullish coalescing seems to have a relatively narrow use case and I'm not sure if the extra cognitive burden of more syntax is worth the terseness, but the terseness does seem to be nice here, so much so that we don't really need the surrounding function anymore, and can probably just inline it.  (I'm not going to do that here though because I don't want to complicate the diff too much).